README.md

```diff
@@ -59,9 +59,13 @@ OpenLayers appreciates contributions of all kinds.  We especially want to thank
 
 See our [GitHub sponsors page](https://github.com/sponsors/openlayers) or [Open Collective](https://opencollective.com/openlayers/contribute/sponsors-214/checkout) if you too are interested in becoming a regular sponsor.
 
-## IntelliSense support and type checking for VS Code
+## Building ES6
```

Comment:
Pretty petty issue, but the ECMAScript 6th edition (ES6) was never published.  Instead it was renamed to ES2015.  Rather than pinning ourselves to a specific version, I think we could use "ES Modules."

And then instead of "Building ES6," since this section is about using the untranspiled sources directly, maybe the heading could be "Using ES Modules" or just "ES Modules."

Reply:
```suggestion
## ES Modules
```